### PR TITLE
insecure doc

### PR DESCRIPTION
**Pull request by yuval-k**

### CODE REVIEW

File: docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+* `relay.cluster` will be the name by which the cluster is referenced in all Gloo Mesh configuration.
+* `relay.serverAddress` is the address by which the Gloo Mesh management plane can be accessed.
+
+Also note that the Enterprise Agent's version should match that of the `enterprise-networking` component running on the
+management cluster. Run `meshctl version` on the management cluster to review the `enterprise-networking` version.
+
+If you haven't already, you can add the repository by running the following:
+
+```shell
+helm repo add enterprise-agent https://storage.googleapis.com/gloo-mesh-enterprise/enterprise-agent
```
Comment (EItanya) [inline docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md:55]: Should we also have them update the `gloo-mesh-enterprise` repo above as well?

Comment (yuval-k) [reply docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md:55]: i think repo update updates all of them

File: docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+service is available to clients outside the cluster, perhaps through a NodePort or ingress solution, and find the address
+before continuing. 
+
+
+Then we will set our variables:
+
+```shell
+CLUSTER_NAME=cluster-2 # Update value as needed
+REMOTE_CONTEXT=kind-cluster-2 # Update value as needed
+ENTERPRISE_NETWORKING_VERSION=<current version> # Update based on meshctl version output
```
Comment (EItanya) [inline docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md:84]: https://github.com/solo-io/gloo-mesh/blame/cert-rotation-api/docs/content/guides/security/vault.md#L202

**Review by EItanya — COMMENTED** [review]
Why are we using helm install here as opposed to meshctl cluster register? This feels like a POC feature which would most likely be used in combination with our CLI tooling

File: docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+{{% notice note %}} Gloo Mesh Enterprise version v1.1.0-beta20 or above is required for this feature. {{% /notice %}}
+{{% notice warning %}} This guide configures Gloo Mesh Enterprise without secure communication. This is not advised for production usage. {{% /notice %}}
+
+This guide will walk you through the basics of registering clusters for management by Gloo Mesh Enterprise using Helm without any secure communication. This is not fit for production, but provides an easy setup, that would allow you to quickly evaluate Gloo Mesh before investing in securing the communication.
+
+## Install Gloo Mesh in Insecure mode
+
+```shell
+MGMT_CONTEXT=kind-cluster-1 # Update value as needed
```
Comment (EItanya) [inline docs/content/setup/cluster_registration/enterprise_insecure_cluster_registration.md:16]: install with `meshctl`?

**Review by EItanya — APPROVED** [review]

**Pull request merged**